### ISSUE TITLE
log_path has an invalid value

### ISSUE THREAD

**Issue by ikeos2** [PaulMcInnis/JobFunnel#58]
# Issue Template

## Description

the log path variable isn't being set. I tried setting the log_path in settings too, but not sure what the issue is. 

## Steps to Reproduce

1. Install via pip
2. Copy settings.yaml into ~/Virtualenv/Lib/site-packages/jobfunnel/config
3. Run 'funnel'

## Expected behavior

Funnel runs

## Actual behavior

```
$ funnel
ConfigError: 'log_path' has an invalid value
```

## Environment

* Build: Latest as of writing
* Operating system and version: Windows 10 using mingw64 w/ python 3.7.3

**Comment by ikeos2** [comment]
If I comment out lines 54 and 55 in config/validate.py, the script will run.

**Comment by studentbrad** [comment]
We will have to try reproducing this issue on Windows 10 using Python 3.7.3.
You'll have to bear with us.  We only have so much time.
Thank you for bringing this to our attention.
In the meantime, if you have any more information let us know here 😄

**Comment by markkvdb** [comment]
Thanks for pointing out this bug! The reason for the ConfigError is that I forgot about Windows users in the validation process 👎 . The regex to check for valid paths did not consider backward slashes (\). This should be fixed in #62 but since I do not have a Windows system could you possibly check whether the changes fixes your problem?

Assuming you have downloaded the repository to your system this can be done by opening the relevant branch and install JobFunnel as

```bash
git checkout origin/logpath-fix
pip install -e .
```

Assuming you are in the `JobFunnel` directory. Let me know whether this works so that we can merge the pull request into the master.

**Comment by markkvdb** [comment]
Solved in #62
